### ISSUE TITLE
org validation failed error

### ISSUE THREAD

**Issue by CGenie** [PoiScript/orgize#69]
Hello,

I got this error from `orgize` when trying to parse bunch of my org files. I'm using version 0.9.0 from crates.io.

```
Org validation failed. 1 error(s) found:
ExpectedChildren { at: NodeId { index1: 21, stamp: NodeStamp(0) } } at DynBlock(DynBlock { block_name: "clocktable", arguments: Some(":maxlevel 2 :scope file :block lastmonth :step month :stepskip0 t :formula \"$4=$2*42.0;t\""), pre_blank: 1, post_blank: 0 })
thread 'main' panicked at ~/.cargo/registry/src/index.crates.io-6f17d22bba15001f/orgize-0.9.0/src/validate.rs:203:17:
Looks like there's a bug in orgize! Please report it with your org-mode content at https://github.com/PoiScript/orgize/issues.
```

**Comment by PoiScript** [comment]
hello, could you try to parse it with version `v0.10.0-alpha.1`?

if it still fails, please provide a minimal reproducible example.

**Comment by CGenie** [comment]
Hello,

Just tested with `v0.10.0-alpha.1` and still I get the error. Minimal org file that I have with this error is:
```org
#+TITLE: test

* First heading

| cat | {3} |
```

The error I'm getting:
```
thread 'main' panicked at /home/przemek/.cargo/registry/src/index.crates.io-6f17d22bba15001f/orgize-0.10.0-alpha.1/src/syntax/object.rs:93:13:
object must have at least three characters: "3}"
```

**Comment by PoiScript** [comment]
@CGenie thanks for the quick response. resolved in `v0.10.0-alpha.2`: 

<img width="874" alt="image" src="https://github.com/PoiScript/orgize/assets/17513314/0b20befd-6322-4dec-9e8e-526723b20ee0">

**Comment by CGenie** [comment]
Thank you, how all my org files work correctly!
